concepts/generators/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for this concept",
-  "authors": ["bethanyg", "cmccandless"],
+  "blurb": "Learn about generators by assigning seats to passengers.",
+  "authors": ["bethanyg", "cmccandless", "J08K"],
```

Comment:
```suggestion
  "authors": ["J08K"],
```

Reply:
No need to have Corey or I as authors unless we contribute significantly to the concept.  Our names are there as placeholders to pass linting.